README.md

```diff
@@ -345,10 +345,28 @@ Call Signatures:
 - `withBrowser.headed(testFn: (context: PleasantestContext) => Promise<void>)`
 - `withBrowser.headed(opts: WithBrowserOpts, testFn: (context: PleasantestContext) => Promise<void>)`
 
-`WithBrowserOpts`:
+`WithBrowserOpts` (all properties are optional):
 
 - `headless`: `boolean`, default `true`: Whether to open a headless (not visible) browser. If you use the `withBrowser.headed` chain, that will override the value of `headless`.
 - `device`: Device Object [described here](https://pptr.dev/#?product=Puppeteer&version=v10.1.0&show=api-pageemulateoptions).
+- `moduleServer`: Module Server options object (all properties are optional)
+  - `plugins`: Array of Rollup, Vite, or WMR plugins to add. They will be applied to files imported through [`utils.runJS`](#pleasantestutilsrunjscode-string-promisevoid) or [`utils.loadJS`](#pleasantestutilsloadjsjspath-string-promisevoid).
+  - `envVars`: Object with string keys and string values for environment variables to pass in as `import.meta.env.*` / `process.env.*`
+  - `esbuild`: [`TransformOptions`](https://esbuild.github.io/api/#transform-api) | `false`: Options to pass to esbuild. Set to false to disable esbuild.
```

Comment:
Are these just applied when using `utils.runJS`/`utils.loadJS`? Or are they run in other scenarios?

Reply:
Only using those functions.

Well actually...
- Hypothetically if you used injectHTML and put in some <script> tags I _think_ it would probably serve them from the module server? But this is not documented and not intended to be used
- If you use loadCSS it uses the module server as well. I am not completely sure how user plugins will interact with rollup-plugin-postcss that is there by default, but I don't think user plugins will end up processing those files (I could be wrong)

Reply:
Gotcha. I ask since you specified the `plugins` only apply in those contexts, but it was less clear for me for the `esbuild` options. Maybe it would be good to add it to that bullet as well?

Or, if all of these properties only affect those methods, maybe that should be moved up to the `moduleServer` bullet point?

Reply:
Yeah I think they should be moved up to moduleServer 👍